### ISSUE TITLE
Beehive crop growth 

### ISSUE THREAD

**Issue by Drakimau** [ProjectEdenGG/Issues#747]
If a beehive is placed with bees attached to it crops nearby would slightly raise the growth chance. 

suggestions on how it could be implemented
- upping the growth chance for all crops in a Y radius of a beehive

**Comment by Blast-MC** [comment]
This sort of already exists within the game by default; from the Wiki:

"Bees can be used to accelerate the growth of crops by pollinating them. After collecting pollen from flowers, bees visually drop pollen particles as they make their way back to their hive or nest. If these particles land on a crop, the crop advances one growth stage. Each bee can pollinate up to 10 crops per trip. Players can utilize this behavior by placing their crops between beehives and flowers to maximize crop pollen exposure."

Do we want to expand upon this and increase the effect or leave it as is?

**Comment by qixils** [comment]
Yes, that mechanic is what warranted Filid's suggestion, as the mechanic is barely noticeable in practice. The proposal is to add a static boost in growth rate for all crops within a 5 block radius of a bee hive.
